File: src/web/lgse/src/app/models/api/user.model.ts
export interface User {
    token?;
    username?;
    userId?;
}

//User List 
export class UserListResponse {
    id : string;
    email: string;
    firstName: string;
    lastName: string;
    roles: [string];
    employeeId: string;
    eusr: string;
    contactNo: string;
    isActive:boolean;
}

export class UserAddRequest {
    firstName: string;
    lastName: string;
    employeeId: string;
    roleId: string;
    eusr: string;
    contactNo: string;
    isActive: boolean;
    email: string;
    IsActiveUser:boolean;
}
export class UserEditRequest
{
    firstName: string;
    lastName: string;
    employeeId: string;
    roleId: string;
    eusr: string;
    contactNo: string;
    isActiveUser: boolean;
    email: string;
    activationOTP:string;
}

export class UserId{
    UserId:string;
}
export class PassUserData
{
    id:string;
    firstName:string;
    lastName:string;
}

File: src/mobile-app/LGSEApp.Services/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LGSEApp.Services.Models
{
    public class UserModel
    {
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string OrgId { get; set; }
        public string EmployeeId { get; set; }
        public string RoleId { get; set; }
        public List<RoleModel> roles { get; set; }
        public string EUSR { get; set; }
        public string ContactNo { get; set; }

    }
    public class UserProfileModel
    {
        public string Email { get; set; }
        public string FirtName { get; set; }
        public string LastName { get; set; }
        public string OrgId { get; set; }
        public string EmployeeId { get; set; }
        public string RoleId { get; set; }
        public List<RoleModel> roles { get; set; }
        public string EUSR { get; set; }
        public string ContactNo { get; set; }

    }
}
